bigquery/samples/export_data_to_cloud_storage.py

```diff
@@ -28,8 +29,10 @@ def export_table(service, cloud_storage_path,
         service: initialized and authorized bigquery
         google-api-client object,
         cloud_storage_path: fully qualified
-        path to a Google Cloud Storage location,
-        e.g. gs://mybucket/myfolder/
+            path to a Google Cloud Storage location,
+            e.g. gs://mybucket/myfolder/
+            export_format: format to export in;
+            "CSV", "NEWLINE_DELIMITED_JSON", or "AVRO".
 
     Returns: an extract job resource representing the
     job, see https://cloud.google.com/bigquery/docs/reference/v2/jobs
```

Comment:
The docstring still has wrong indentations.

It should be as follows:
```python
def export_table(service, cloud_storage_path,
                 projectId, datasetId, tableId,
                 export_format="CSV",
                 num_retries=5):
    """
    Starts an export job.

    Args:
        service: Initialized and authorized bigquery
            google-api-client object.
        cloud_storage_path: Fully qualified
            path to a Google Cloud Storage location.
            e.g. gs://mybucket/myfolder/
        export_format: Format to export in;
            "CSV", "NEWLINE_DELIMITED_JSON", or "AVRO".

    Returns: An extract job resource representing the
        job, see https://cloud.google.com/bigquery/docs/reference/v2/jobs
    """
```

* Use period at the end of the clause
* Use capitalized first letter
* Correct indentation (continuation line should have deeper indent, for example, the second line for the "service" argument should have deeper indentation)